pkg/kubelet/util/cache/version_cache.go

```diff
@@ -17,14 +17,12 @@ limitations under the License.
 package cache
 
 import (
-	"fmt"
 	"sync"
 	"time"
 
 	"github.com/golang/glog"
 
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
-	"k8s.io/kubernetes/pkg/util/wait"
 )
 
 type VersionCache struct {
```

Comment:
IIUC, this will be a generic with a expiration policy. Are there any existing library that we can reuse? I didn't look deeply, but https://github.com/kubernetes/kubernetes/blob/master/pkg/client/cache/expiration_cache.go seems like a candidate.